packages/form/package.json

```diff
@@ -35,6 +35,7 @@
     "ember-math-helpers": "^4.0.0",
     "ember-power-select": "^7.2.0",
     "ember-resources": "^6.4.2",
+    "ember-truth-helpers": "^4.0.3",
```

Comment:
As we only need this in one place I'd suggest putting the logic into the JS file to avoid unnecessary dependencies

Reply:
I totally see your point, but I wanted to spare the `component.js` file (it's now there anyways) and saw that the `workflow` package has it as well. Further I think the [`action-button.hbs`](https://github.com/projectcaluma/ember-caluma/blob/649c5d8524bf930a78c9395e1516b38b63c62316/packages/form/addon/components/cf-field/input/action-button.hbs#L33) is using it as well, or am I missing something there?

Reply:
True. There are some other places where it's used as well. I withdraw my comment :wink:

Reply:
**But:** remember to add it in the blueprint as the host app must install it because it's used in templates

Reply:
`done`